docs/cockroach/README.md

```diff
@@ -8,3 +8,5 @@
 1. Create a sql user. Set it to anything other than `root`.
    ![](./cockroach-create-sql-user.png)
 1. Save connection string. You don't need to install the CA root cert. See [link](https://www.cockroachlabs.com/docs/cockroachcloud/serverless-faqs.html#what-certificates-do-i-need-to-connect-to-my-cluster)
+   - The `DATABASE_URL` environment variable in Starter Kit should be set to this connection string
+1. Run `npm run migrate` on the newly-created database
```

Comment:
nit: technically no need, since `prebuild` script will run it.

Reply:
Right, I didn't see this. Thanks for the catch.